tools.yml

```diff
@@ -2342,29 +3255,43 @@ tools:
     mem: 20
   toolshed.g2.bx.psu.edu/repos/iuc/vardict_java/vardict_java/.*:
     cores: 2
-    mem: 128
+    mem: 36.5
```

Comment:
we had a reason for this ... if I would just remember. A rule is probably needed here.